examples/custom-receiver/package.json

```diff
@@ -20,7 +20,7 @@
     "koa": "^3"
   },
   "devDependencies": {
-    "@types/koa__router": "^12",
+    "@types/koa": "^3.0.1",
```

Comment:
👁️‍🗨️ note: Noticed that we can perhaps prefer the `koa` typings here!

> @koa/router is written in TypeScript and includes comprehensive type definitions out of the box. No need for `@types/* `packages!

🔗 https://github.com/koajs/router?tab=readme-ov-file#typescript-support